support/schema/bashly.yml

```diff
@@ -335,6 +335,31 @@ definitions:
           - ~/config.ini
     patternProperties: *custom-properties
     additionalProperties: false
+  variable:
+    title: variable
+    description: |-
+      A global bash variable
+      https://bashly.dannyb.co/configuration/command/#variables
+    type: object
+    required:
+      - name
+    properties:
+      name:
+        title: name
+        description: A name for the current variable
+        type: string
+        minLength: 1
+        examples:
+          - output_dir
+      value:
+        title: value
+        description: A value for the current variable
+        examples:
```

Comment:
What if the variable value is `null` ? Is it an error? What it corresponds to in Bash (in the context of Bashly) if it's not?